src/app/pages/wishlist/wishlist.component.ts

```diff
@@ -25,4 +35,9 @@ export class WishlistComponent implements OnInit {
         this.wishlistService.removeWishProduct(id).subscribe(products => this.wishProducts = products);
     }
 
+    ngOnDestroy(): void {
+        this.unsubscribeAll.next();
+        this.unsubscribeAll.complete();
+    }
```

Comment:
You haven't unsubscribed... Fill this variable with subscriptions using pipe `takeUntil()`

[Just to wide your knowledge](https://blog.bitsrc.io/6-ways-to-unsubscribe-from-observables-in-angular-ab912819a78f)